lms/djangoapps/verify_student/management/commands/tests/test_verify_student.py

```diff
@@ -64,3 +68,38 @@ def test_retry_failed_photo_verifications(self):
         call_command('retry_failed_photo_verifications')
         attempts_to_retry = SoftwareSecurePhotoVerification.objects.filter(status='must_retry')
         assert not attempts_to_retry
+
+    def test_args_from_database(self):
+        """Test management command arguments injected from config model."""
+        # Nothing in the database, should default to disabled
+        with self.assertRaisesRegex(CommandError, 'SSPVerificationRetryConfig is disabled*'):
+            call_command('retry_failed_photo_verifications', '--args-from-database')
+
+        # Add a config
+        config = SSPVerificationRetryConfig.current()
+        config.arguments = '--verification-ids 1,2,3'
+        config.enabled = True
+        config.save()
+
+        with patch('lms.djangoapps.verify_student.models.requests.post', new=mock_software_secure_post_error):
+            self.create_and_submit("RetryRoger")
+
+            with LogCapture(LOGGER_NAME) as log:
+                call_command('retry_failed_photo_verifications')
+
+                log.check_present(
```

Comment:
you should also check it was done successfully.

Reply:
This case is already tested in above test https://github.com/edx/edx-platform/blob/master/lms/djangoapps/verify_student/management/commands/tests/test_verify_student.py#L63